doc/build/sysbuild/index.rst

```diff
@@ -0,0 +1,503 @@
+flashes both the ``mcuboot`` and ``hello_world`` application images to the
+board.
+
+More detailed information regarding the use of MCUboot with Zephyr can be found
+in the `MCUboot with Zephyr`_ documentation page on the MCUboot website.
+
+.. note::
+
+   MCUBoot default configuration will perform a full chip erase when flashed.
+   This can be controlled through the MCUBoot Kconfig option
+   ``CONFIG_ZEPHYR_TRY_MASS_ERASE``. If this option is enabled, then flashing
+   only MCUBoot, for example using ``west flash --domain mcuboot``, may erase
+   the entire flash, including the main application image.
+
+Sysbuild Kconfig file
+*********************
+
+You can set sysbuild's Kconfig options for a single application using
+configuration files. By default, sysbuild looks for a configuration file named
+``sysbuild.conf`` in the application top-level directory.
+
+In the following example, there is a :file:`sysbuild.conf` file that enables building and flashing with
+MCUboot whenever sysbuild is used:
+
+.. code-block:: none
+
+   <home>/application
+   ├── CMakeLists.txt
+   ├── prj.conf
+   └── sysbuild.conf
+
+
+.. code-block:: none
+
+   SB_CONFIG_BOOTLOADER_MCUBOOT=y
+
+You can set a configuration file to use with the
+``-DSB_CONF_FILE=<sysbuild-conf-file>`` CMake build setting.
+
+For example, you can create ``sysbuild-mcuboot.conf`` and then
+specify this file when building with sysbuild, as follows:
+
+.. tabs::
+
+   .. group-tab:: ``west build``
+
+      .. zephyr-app-commands::
+         :tool: west
+         :app: samples/hello_world
+         :board: reel_board
+         :goals: build
+         :west-args: --sysbuild
+         :gen-args: -DSB_CONF_FILE=sysbuild-mcuboot.conf
+         :compact:
+
+   .. group-tab:: ``cmake``
+
+      .. zephyr-app-commands::
+         :tool: cmake
+         :app: share/sysbuild
+         :board: reel_board
+         :goals: build
+         :gen-args: -DAPP_DIR=samples/hello_world -DSB_CONF_FILE=sysbuild-mcuboot.conf
+         :compact:
+
+.. _sysbuild_zephyr_application:
+
+Adding Zephyr applications to sysbuild
+**************************************
+
+You can use the ``ExternalZephyrProject_Add()`` function to add Zephyr
+applications as sysbuild domains. Call this CMake function from your main
+:file:`CMakeLists.txt` file, or any other CMake file you know will run.
+
+Targeting the same board
+========================
+
+To include ``my_sample`` as another sysbuild domain, targeting the same board
+as the main image, use this example:
+
+.. code-block:: cmake
+
+   ExternalZephyrProject_Add(
+     APPLICATION my_sample
+     SOURCE_DIR <path-to>/my_sample
+     BUILD_ALWAYS True
+   )
+
+This could be useful, for example, if your board requires you to build and flash an
+SoC-specific bootloader along with your main application.
+
+Targeting a different board
+===========================
+
+In Zephyr a board may refer to:
```

Comment:
"In Zephyr a board may refer to:"
This is always the case, in device tree for instance, boards and socs are different notions.
Can you replace by "For SysBuild", or "In Zephyr build system" ?

Reply:
Done